images/Dockerfile

```diff
@@ -1,24 +1,12 @@
 # Docker image for running E2E tests using Argo.
 
-FROM python:2.7-slim
+FROM ubuntu:xenial
```

Comment:
just curious: why do we change to xenial?

Reply:
python2.7 was a vestige of the Airflow image we were originally. Using.
It makes more sense to use a generic linux image.